cluster/manifests/stackset-controller/rbac.yaml

```diff
@@ -99,6 +99,18 @@ rules:
   - create
   - update
   - patch
+{{- if eq .Cluster.ConfigItems.stackset_configmaps_support_enabled "true" }}
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - list
+  - create
+  - update
+  - delete
```

Comment:
I don't think we need `delete` anymore. Maybe let's just align it with the same definition as for events to keep it simple:
```
  - get
  - list
  - watch
  - create
  - update
  - patch
  ```